### PR TITLE
Add support for Amazon Linux 2023 in Vulnerability Detector

### PR DESCRIPTION
**Pull request by Leoquicenoz**
## Description

|Related issue|
|---|
|#6366|

With the addition of a new version (`Amazon Linux 2023`) to the **Amazon Linux** support in the **Vulnerability Detector** module, **Wazuh**'s documentation needs to be updated to include references about this new OS version in the correspondent pages.

The main points to cover include:

- [x] The Vulnerability Detector section must include the new feed and the Amazon Linux 2023 as a supported platform
- [x] All vulnerability-related usage guides must be updated with information and configuration examples for the new OS version. 

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns.

### CODE REVIEW

File: source/user-manual/reference/ossec-conf/vuln-detector.rst [hunk 1]
```diff
@@ -180,8 +180,10 @@ Configuration block to specify vulnerability updates.
...
 |                    |                           |                    |              +---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 |                    |                           |                    | alas         | amazon-linux-2 / 2                                                                                                                                                                                                                                                        |
+|                    |                           |                    |              +---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```
Comment (mateocervilla) [inline source/user-manual/reference/ossec-conf/vuln-detector.rst:183]:
```suggestion
|                    |                           |                    |              +---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

File: source/user-manual/reference/ossec-conf/vuln-detector.rst [hunk 2]
```diff
@@ -180,8 +180,10 @@ Configuration block to specify vulnerability updates.
...
 |                    |                           |                    | alas         | amazon-linux-2 / 2                                                                                                                                                                                                                                                        |
+|                    |                           |                    |              +---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+   
+|                    |                           |                    |              | amazon-linux-2022 / 2022                                                                                                                                                                                                                                                          |
```
Comment (mateocervilla) [inline source/user-manual/reference/ossec-conf/vuln-detector.rst:184]:
```suggestion
|                    |                           |                    |              | amazon-linux-2022 / 2022                                                                                                                                                                                                                                                  |
```

**Review by mateocervilla — CHANGES_REQUESTED** [review]

**Review by MarcelKemp — CHANGES_REQUESTED** [review]
Add what is supported and missing from Amazon Linux in the following files:
- `source/deployment-options/deploying-with-ansible/reference.rst`
- `source/user-manual/capabilities/vulnerability-detection/configuring-scans.rst`
- `source/user-manual/capabilities/vulnerability-detection/how-it-works.rst`

**Review by MarcelKemp — APPROVED** [review]

**Review by javimed — COMMENTED** [review]
What if we also add a local path in this sample offline configuration:

- https://github.com/wazuh/wazuh-documentation/pull/6446/files#diff-bb9d48338874e2ba0c70be0cfd88ba34a0eef3e863218c947ee87305845238d1R509-R510

**Review by javimed — APPROVED** [review]

**Pull request merged**